runtime/lua/vim/img/_terminal.lua

```diff
@@ -0,0 +1,140 @@
+---@class vim.img.terminal
+---@field private __tty_name string
+local M = {}
+
+local TERM_CODE = {
+  BEL = '\x07', -- aka ^G
+  ESC = '\x1B', -- aka ^[ aka \033
```

Comment:
The `\x` prefix is a LuaJIT extension and won't work in PUC Lua. You need to use decimal notation, i.e. `\007` for BEL and `\027` for ESC.